### PR TITLE
Feature/i18n

### PR DESCRIPTION
**Pull request by b-studios**
In this feature branch we implement support for internationalization.

### CODE REVIEW

**Review by jiribenes — CHANGES_REQUESTED** [review]
Hi, @b-studios, thanks for your contribution!

I18n is really useful for us since with this change, we can add support for other languages — especially useful for the international students!

The code looks alright, but could you maybe add an exclamation mark at the end of the strings? 
We want the students to be *super excited* about running a Hello World! project.

**Comment by b-studios** [comment]
Well, I don't like exclamation marks.

**Comment by jiribenes** [comment]
OK, fair.

**Review by jiribenes — APPROVED** [review]
LGTM, thanks

File: hello.scala [hunk 1]
```diff
@@ -1,8 +1,18 @@
 import scala.io.StdIn.readLine
 
+trait Messages {
+  val welcome: String
+  def greeting(os: String, jdk: String): String
+}
+
+object EN extends Messages {
+  val welcome = "Hey, what's your name?! (please press <ENTER> after entering your name)"
```
Comment (jiribenes) [inline hello.scala:9]: Good job adding an exclamation mark!

**Pull request merged**